distributed/scheduler.py

```diff
@@ -201,11 +201,12 @@ def versions(self):
         return self._versions
 
 
+@cclass
 class WorkerState:
     """
     A simple object holding information about a worker.
 
-    .. attribute:: address
+    .. attribute:: address: str | int
```

Comment:
I'm surprirsed that this can be an int.  I would expect the variation to exist here for name, but address I would always expect to be a string today.

Reply:
Moving to this thread ( https://github.com/dask/distributed/pull/4294/files#r535542496 )